runtime/queries/column_timeseries.go

```diff
@@ -4,17 +4,23 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"math/big"
 	"strconv"
 	"time"
 
 	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
 	"github.com/rilldata/rill/runtime"
 	"github.com/rilldata/rill/runtime/drivers"
+	"google.golang.org/protobuf/types/known/structpb"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 const IsoFormat string = "2006-01-02T15:04:05.000Z"
 
+type ColumnTimeseriesResult struct {
+	Meta []*runtimev1.MetricsViewColumn
+	Data *runtimev1.TimeSeriesResponse
```

Comment:
To ensure `Data` is never nil when other code tries to access one of its members, should we flatten it out here to:
```go
	Results    []*TimeSeriesValue  
	Spark      []*TimeSeriesValue 
	TimeRange  *TimeSeriesTimeRange
	SampleSize int32  
```
and then build the `*runtimev1.TimeSeriesResponse` in the API handler instead? 

Reduces chance of `nil` pointer access